packages/frontend/src/components/wallet-migration/modals/CleanKeysModal/AccessKeyList.jsx

```diff
@@ -0,0 +1,127 @@
+                text-overflow: ellipsis;
+            }
+
+            .keys-to-remove {
+                color: #de2e32;
+            }
+        }
+
+        .expand-keys {
+            flex: 1;
+            padding: 16px 0 0 0;
+        }
+    }
+`;
+
+const AccessKeyList = ({ account, onClose, onNext, selectKey, selectedKeys }) => {
+    const [expanded, setExpanded] = useState(true);
+    return (
+        <AccessKeyListContainer>
+            <h3 className='title'>
+                <Translate id='walletMigration.cleanKeys.accountTitle' />
+            </h3>
+            <div
+                className={classNames(['account', expanded ? 'open' : ''])}
+                id='full-access-keys'
+                onClick={() => setExpanded(!expanded)}
+            >
+                <div className='account-detail'>
+                    <p className='account-id'>{account.accountId}</p>
+                    <p className='keys-to-remove'>
+                        {account.accessKeys.length}/{account.totalAccessKeys}
+                        <Translate id='walletMigration.cleanKeys.fullAccessKeys' />
+                    </p>
+                </div>
+                <div className='expand-keys'>
+                    <ChevronIcon color='#0072ce' />
+                </div>
+            </div>
+            <Accordion
+                trigger='full-access-keys'
+                className='breakdown'
+            >
+                {account.accessKeys.map(({ kind, publicKey }) => (
+                    <div className='access-key' key={publicKey}>
+                        <div className='public-key'>
+                            <span className='key-prefix'>
+                                {publicKey.replace('ed25519:', '').slice(0, 16)}
+                            </span>
+                            &nbsp;
+                            (<Translate id={`walletMigration.cleanKeys.keyTypes.${kind}`} />)
+                        </div>
+                        <div className='remove-checkbox' onClick={() => selectKey(publicKey)}>
+                            <Checkbox checked={selectedKeys[publicKey]} />
+                        </div>
+                    </div>
+                ))}
+            </Accordion>
+            <ButtonsContainer vertical>
+                <StyledButton
+                    onClick={onNext}
+                    fullWidth
+                    disabled={!Object.values(selectedKeys).some((remove) => remove)}
+                    data-test-id="cleanupKeys.continue"
+                >
+                    <Translate id='walletMigration.cleanKeys.removeKeys' />
+                </StyledButton>
+                <StyledButton
+                    className='gray-blue'
+                    onClick={onClose}
+                    fullWidth
+                >
+                    <Translate id='button.cancel' />
+                </StyledButton>
+            </ButtonsContainer>
+        </AccessKeyListContainer>
```

Comment:
![image](https://user-images.githubusercontent.com/6027014/202340693-33fba80c-6de7-4429-91c3-0704748e6ed4.png)
Some weird css issue here

Reply:
hmm I tried to solve this UI issue by myself but couldn't get it around with using `Accordion` component. (I feel that if we want to solve this, we may better to write UI newly)

Either way, I will sort this out later